packages/hash/api/package.json

```diff
@@ -44,8 +46,8 @@
     "express": "4.17.1",
     "express-session": "1.17.2",
     "graphql": "15.5.1",
+    "graphql-scalars": "1.17.0",
     "graphql-tag": "2.12.5",
-    "graphql-type-json": "0.3.2",
```

Comment:
I swapped this dependency to `graphql-scalars` which is better maintained and contains more scalar types, if we need them. `graphql-type-json` was last updated in June 2020.